readium/streamer/src/main/java/org/readium/r2/streamer/parser/audio/AudioParser.kt

```diff
@@ -79,7 +80,7 @@ public class AudioParser(
 
         val readingOrderWithFormat =
             asset.container
-                .mapNotNull { url -> entryFormats[url]?.let { url to it } }
```

Comment:
I think we might want to keep the original URLs here, as we do with other formats.

Reply:
We do, this is used to get the `Format` and it doesn't replace the `Url` of the container.

Although it's not strictly necessary here, the `Url` should be the same instances.

Reply:
Ok, I read too quick.